strings-csharp/HexToByteArray/Tests/Test.cs

```diff
@@ -4,85 +4,66 @@ namespace Tests
 {
     public class Test
     {
-        private readonly ConvertHex _convertHex;
         private const string HexString = "FF008000";
-        public Test()
-        {
-            _convertHex = new ConvertHex();
-        }
+        private readonly byte[] _expectedResult = { 255, 0, 128, 0 };
+        private readonly byte[] _expectedEmptyResult = new byte[0];
+
 
         [Fact]
         public void WhenValidHexPassed_ThenConverionSuccessful()
         {
-            byte[] expectedResult = { 255, 0, 128, 0 };
-            var result = _convertHex.FromHexString(HexString);
+            var result = ConvertHex.FromHexString(HexString);
 
-            Assert.Equal(expectedResult, result);
+            Assert.Equal(_expectedResult, result);
         }
 
         [Fact]
         public void WhenInvalidFormatHexPassed_ThenConverionFail()
         {
-            var expectedResult = "FormatException";
-            try
-            {
-                var result = _convertHex.FromHexString("Hello");
-            }
-            catch (Exception ex)
-            {
-                Assert.Equal(expectedResult, ex.GetType().Name);
-            }
+            var ex = Assert.Throws<FormatException>(() => ConvertHex.FromHexString("Hello"));
+
+            Assert.Equal("FormatException", ex.GetType().Name);
         }
 
         [Fact]
         public void WhenNullatHexPassed_ThenConverionFail()
         {
-            var expectedResult = "ArgumentNullException";
-            try
-            {
-                var result = _convertHex.FromHexString("");
-            }
-            catch (Exception ex)
-            {
-                Assert.Equal(expectedResult, ex.GetType().Name);
-            }
+            string? nullHexString = null;
+            var ex = Assert.Throws<ArgumentNullException>(() => ConvertHex.FromHexString(nullHexString));
+
+            Assert.Equal("ArgumentNullException", ex.GetType().Name);
```

Comment:
Same as previous, we don't need to check the type as the Throws<T> already did.
Related to that, just for learning experience. A cleaner way to write a test that needs to verify a certain type would be to use the Assert.IsType<T>() assertion.